deploy/openshift/templates/README.md

```diff
@@ -0,0 +1,155 @@
+## Context
+
+All commands are executed in the directory where this README is located.
+
+## Templates and Parameters
+
+Templates are provided with a set of predefined params which you can override with with `-p key=value`. If you miss envs and parameters, you can add them to your template.
+
+IMPORTANT! **ROUTING_SUFFIX** value in the below commands `$(minishift ip).nip.io` works for MiniShift only. You MUST provide **own routing suffix** when deploying to OCP cluster.
+
+## Why many templates?
+
+Since there are several Che flavors and configuration options, it is not possible to list all objects in one template, while duplicating templates makes it more difficult to maintain them.
+
+## Che server logs persistence
+
+All of the below commands create a PVC for Che server and Che deployment uses this PVC.
+
+It is possible to avoid using PVC for Che server - just skip creating pvc `oc apply -f pvc/che-server-pvc.yaml` and `oc set volume dc/che --add -m /data --name=che-data-volume --claim-name=che-data-volume` commands.
```

Comment:
Actually, PVC is used by Single-User Che to store H2 Database file. Please, add a note that without PVC Single-User Che will lose all data after Pod restarting.

Reply:
Good point will do. For single user PVC is created by default - this is what I suggest in an example command.

But once again - good point.